.github/actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -235,10 +236,57 @@ When contributing to Jetpack, we have [a few suggestions](https://github.com/Aut
 
 This comment will be updated as you work on your PR and make changes. If you think that some of those checks are not needed for your PR, please explain why you think so. Thanks for cooperation :robot:
 
+******`;
+
+	// If some of the tests are failing, display list of things that could be updated in the PR description to fix things.
+	const recommendations = `
+${
+	! hasLongDescription
+		? `Please edit your PR description and explain what functional changes your PR includes, and why those changes are needed.`
+		: ''
+}
+${
+	! hasPrivacy
+		? `We would recommend that you add a section to the PR description to specify whether this PR includes any changes to data or privacy, like so:
+\`\`\`
```

Comment:
Totally optional, but if you want to avoid all the backslashes you can instead use 3 or more tildes to delimit code blocks.
https://github.github.com/gfm/#fenced-code-blocks

Reply:
Ah, TIL, thank you. That's cleaner. Done in 8c1462ffa38f2128af1bcadffe29773797f1201c